Add xditya/GrammyCloneableBot (#36)
grammyjs/awesome-grammY: README.md

- [waptik/grammy-on-deta-template](https://github.com/waptik/grammy-on-deta-template) - A template to start a grammY bot and deploy it on [Deta](https://www.deta.sh). [ᴜsᴇ](https://t.me/GrammyOnDetaBot)
- [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram Web App starter template.
- [Borodutch/telegram-bot-starter](https://github.com/Borodutch/telegram-bot-starter) - Telegram bot starter based on grammY.

> - [xditya/GrammyCloneableBot](https://github.com/xditya/GrammyCloneableBot) - A template for a clone-able telegram bot, i.e., one which allows users to run their own instance of the bot within your bot.

Reviewer: Do you have any other idea for the description that is as clear as the rest, or you like to see mine?

Author: You can suggest, would be great!

Reviewer: ```suggestion
- [xditya/GrammyCloneableBot](https://github.com/xditya/GrammyCloneableBot) - A template for bots that can be cloned.
```
